pushbullet

```diff
@@ -28,6 +28,7 @@ push - Pushes data to a device. (the device name can simply be a unique part of
        the name that \"list\" returns)
 pushes active - Get your 'active' pushes (pushes that haven't been deleted) 
 delete $iden - Deletes a push, you must give the push 'iden', see https://docs.pushbullet.com/v2/pushes/
+delete old $number - Deletes all pushes except the last $number
```

Comment:
how about naming this "except"? You also used this word in the title and it sounds more natural.
"pusbullet delete except 3" instead of "pushbullet delete old 3".

To also to have the $number displayed in the help text the $ has to be escaped (yes I know I introduced this error in the line above, did not yet open a pull request to correct this).